src/components/CategoryFilter.js

```diff
@@ -0,0 +1,23 @@
+import { Select } from "./Select.js";
+
+export class CategoryFilter {
+  #select;
+
+  constructor({ id, name, options, onChange }) {
+    this.#select = new Select({
+      id,
+      name,
+      options,
+      onChange,
+      className: "restaurant-filter",
+    });
+  }
```

Comment:
```suggestion
  constructor(props) {
    this.#select = new Select({
      ...props,
      className: "restaurant-filter",
    });
  }
```

이렇게 props로 묶어주면 어떨까요!?